### DISCUSSION TITLE
希望能有个撤回开关……

### DISCUSSION THREAD

**Discussion by SnowMoonSS** [Samarium150/mirai-console-lolicon#6] [Ideas]
希望能有个撤回开关……通过配置文件配置什么的……

**Comment by Samarium150** [comment]
> 希望能有个撤回开关……通过配置文件配置什么的……

可以做到。一开始没有设置这个选项是因为R18的图大概率能把群/bot炸了。而且现在没有实现权限的分配，谁都能通过命令修改配置。等我研究一下再安排

**Reply by SnowMoonSS** [reply]
> > 希望能有个撤回开关……通过配置文件配置什么的……
> 
> 可以做到。一开始没有设置这个选项是因为R18的图大概率能把群/bot炸了。而且现在没有实现权限的分配，谁都能通过命令修改配置。等我研究一下再安排

感谢！

**Reply by Samarium150** [reply]
> > > 希望能有个撤回开关……通过配置文件配置什么的……
> > 
> > 
> > 可以做到。一开始没有设置这个选项是因为R18的图大概率能把群/bot炸了。而且现在没有实现权限的分配，谁都能通过命令修改配置。等我研究一下再安排
> 
> 感谢！

已经在 #19 中实现

**Reply by SnowMoonSS** [reply]
好耶
